Mathlib/Topology/Homotopy/Basic.lean

```diff
@@ -360,12 +360,10 @@ The type of homotopies between `f₀ f₁ : C(X, Y)`, where the intermediate map
 `P : C(X, Y) → Prop`
 -/
 structure HomotopyWith (f₀ f₁ : C(X, Y)) (P : C(X, Y) → Prop) extends Homotopy f₀ f₁ where
+  -- porting note: todo: use `toHomotopy.curry t`
   /-- the intermediate maps of the homotopy satisfy the proprerty -/
```

Comment:
Typo seems to be here and not fixed.

Reply:
I expanded the description. Is there one more typo here? If yes, could you please fix it?

Reply:
```suggestion
  /-- the intermediate maps of the homotopy satisfy the property -/
```